client/reader/notifications/controller.jsx

```diff
@@ -44,14 +44,32 @@ export function notifications( context, next ) {
 	context.primary = (
 		<>
 			<NotificationTitle />
-			<div className="reader-notifications__panel">
-				<AsyncLoad
-					require="calypso/notifications"
-					isShowing
-					checkToggle={ () => {} }
-					placeholder={ null }
-					isGlobalSidebarVisible={ shouldShowGlobalSidebar }
-				/>
+			<div className="reader-notifications__page">
+				<div className="reader-notifications__page-notice">
+					<p>
+						{ translate(
+							"Didn't expect to see this page? {{learnMoreLink}}Learn about 3rd party cookies.{{/learnMoreLink}}",
+							{
+								components: {
+									learnMoreLink: (
+										<a
+											href={ localizeUrl( 'https://wordpress.com/support/third-party-cookies/' ) }
```

Comment:
Does the article linked here work to provide enough context for users?
https://wordpress.com/support/third-party-cookies/

Reply:
I think a new support doc page explaining what's going on with the notification panel specifically would be better, but this doc could work in the meantime.

Reply:
Ok pushing as is sounds good, and I or someone else can follow up with a new support doc and link change.